docs/docs/rest-api/public/api/v2/types/podStatus.raml

```diff
@@ -191,6 +191,10 @@ types:
         type: datetime
         description: |
           Time that this status was last modified (some aspect of status did change)
+      role:
```

Comment:
This is what breaks the UpgradeIntegrationTest. The previous versions of marathon don't return this field, therefore the deserialisation of the Status from the older version fails. 
Either we make this optional, or access the raw json in the Test :-/

Reply:
I'm inclined to not make it optional just because of a test. PodStatus will always return the field.

Reply:
Yeah, Aleksey fixed it :)